openlibrary/templates/lib/nav_head.html

```diff
@@ -61,7 +61,7 @@
   <ul class="navigation-component">
     <li class="browse-menu">
       <a>$_('Browse') <img class="down-arrow"
-        width="7" height="4" aria-hidden="true" alt="$_('Browse')" role="presentation" src="/static/images/down-arrow.png"/></a>
+        width="7" height="4" aria-hidden="true" alt="$_('Browse')" src="/static/images/down-arrow.png"/></a>
```

Comment:
Role presentation actually looks correct here. We likely don't want these to have an `alt` text, since they'll be read twice by screenreaders.

```suggestion
        width="7" height="4" aria-hidden="true" src="/static/images/down-arrow.png" alt="" role="presentation"/></a>
```

Src: 
- https://www.w3.org/WAI/tutorials/images/decorative/
- https://www.w3.org/WAI/tutorials/images/functional/
- https://www.w3.org/WAI/tutorials/menus/flyout/

Ditto for the other icons that appear right next to text with the same label.

Reply:
Thank you for reviewing, will be pushing the changes in a bit.